apps/sla/time.py

```diff
@@ -201,3 +227,13 @@ def skip_week_ending(dt):
         dt = add_days(dt, 1)
 
     return dt
+
+
+def skip_shutdown_idle_days(dt):
+    """
+    if the given date falls on a non-business day or a shutdown idle day move it to the next business day
+    """
+    while not is_business_day(dt) or (dt.month, dt.day) in SHUTDOWN_IDLE_DAYS:
+        dt = add_days(dt, 1)
+
+    return skip_week_ending(dt)
```

Comment:
Aren't the "_no week ending_" and "_no shutdown_" options independent? With this, we are enforcing the first into the second even if it wasn't configured to do so. Is this intended?